src/scss/_main.scss

```diff
@@ -1005,9 +1010,17 @@ nav.bd-links {
         > a {
             font-size: 0.9em;
         }
+        .fa-chevron-up, .fa-chevron-down {
+            float: right;
+            font-size: 1em;
+            margin-top: 0.4em;
+            margin-right: 0.4em;
```

Comment:
```suggestion
            font-size: 0.75em;
            margin-top: 0.5em;
            margin-right: 1.25em;
```

Before:

<img width="281" alt="Screenshot 2020-10-05 at 7 48 37 PM" src="https://user-images.githubusercontent.com/3275593/95091816-88400680-0744-11eb-808f-346dc9a8c776.png">

After:

<img width="281" alt="Screenshot 2020-10-05 at 7 50 53 PM" src="https://user-images.githubusercontent.com/3275593/95091823-8a09ca00-0744-11eb-9ddd-50efbeb6d343.png">

You might even want to use 1.5em for the right margin, although the spacing when the scrollbar is not hidden/overlaid.